pom.xml

```
@@ -553,9 +553,9 @@
             <build>
                 <plugins>
                     <plugin>
-                        <groupId>org.jacoco</groupId>
+                        <groupId>za.co.absa.jacoco</groupId>
                         <artifactId>jacoco-maven-plugin</artifactId>
-                        <version>0.8.8</version>
+                        <version>0.8.9</version>
```

What I meant was to just use the latest version of the absa fork
```suggestion
                        <version>0.8.10</version>
```

---

Sure, I have found not finished push on my local. Solved. Now you should see usage of latest version of Jacoco Absa fork.